css/about.css

```
@@ -0,0 +1,698 @@
+}
+
+.contact a {
+  font-weight: 800;
+  font-size: 15px;
+  color: #fff;
+  text-decoration: underline;
+  text-transform: lowercase;
+}
+
+.sub__navbar {
+  width: 100%;
+  height: 40px;
+  gap: 50px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: #272a31;
+}
+
+.socialIcon {
+  display: flex;
+  gap: 50px;
+  margin-left: 20px;
+}
+
+.socialIcon a {
+  font-size: 25px;
+  color: #fff;
+}
+
+.program a {
+  display: none;
+}
+
+.figure1 a,
+.figure2 a {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  gap: 20px;
+  background: #ec534296;
+  color: #eff2fa;
+  transition: 0.5s ease;
+}
+
+.sub__navMenu ul li a {
+  color: #fff;
+  transition: 0.3s;
+}
+
+.main__navMenu ul li a {
+  color: #272a31;
+  font-weight: 500;
+  transition: 0.3s ease;
+}
+
+.socialIcon a:hover {
+  color: #ec5242;
+  transition: 0.3s;
+}
+
+.sub__navMenu ul {
+  display: flex;
+  gap: 30px;
+}
+
+.figure1 a:hover,
+.figure2 a:hover {
+  border-radius: 0;
+  background: #ec534296;
+  box-shadow:
+    -11px 11px 25px #5a5a5a,
+    11px -11px 25px #fff;
+}
+
+.sub__navMenu ul li a:hover {
+  color: #ec5242;
+}
+
+.main__navbar {
+  height: 600px;
+  width: 70%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-around;
+  background: #fff;
+}
+
+.main__navMenu ul {
+  height: 400px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.close button,
+.more button {
+  padding: 20px;
+  width: 100%;
+  background: #fff;
+  border: 2px solid #d3d3d3;
+  transition: 0.3s;
+}
+
+.main__navMenu ul li button {
+  width: 150px;
+  height: 50px;
+  border: 5px solid #ec5242;
+  background: #fff;
+  color: #ec5242;
+  transition: 0.3s ease-in-out;
+}
+
+.more button:hover,
+.close button:hover {
+  background: var(--black);
+  color: var(--grey);
+}
+
+.main__navMenu ul li a:hover {
+  color: #ec5242;
+}
+
+.main__navMenu ul li button:hover {
+  color: #fff;
+  background: #d3d3d3;
+}
+
+/* -----------------------------Navbar Section Ends------------------------------------------------------ */
+
+/* -----------------------------About banner Section Starts---------------------------------------------- */
+
+.headerBanner {
+  height: 500px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  padding: 16px;
+  gap: 7px;
+  background-image: url(/Module1--CapstoneProject/img/1025341.png);
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.banner {
+  width: 90%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  text-align: center;
+}
+
+.subHeader {
+  color: #ec5242;
+  font-weight: 500;
+  font-size: 24px;
+}
+
+.mainHeader {
+  font-weight: 800;
+  font-size: 33px;
+  background-image: url(/Module1--CapstoneProject/img/water.jpeg);
```

- [x] And, finally, here: 🤓 

```suggestion
  background-image: url(/img/water.jpeg);
```